### PR TITLE
Updated Copier from template

### PR DESCRIPTION
**Pull request by etobella**
From Travis to GH Actions

### CODE REVIEW

**Comment by etobella** [comment]
@ernestotejeda Some of your modules have problems with the development status. Some of them comes from other repos (like sale_order_secondary_unit). I will propose the promotion for all modules...

**Review by olgamarcocb — APPROVED** [review]
LGTM :+1:

File: base_edi/__manifest__.py [hunk 1]
```diff
@@ -5,8 +5,8 @@
 {
     "name": "Base EDI",
     "summary": """Base module to aggregate EDI features.""",
-    "version": "13.0.1.0.2",
-    "development_status": "Alpha",
+    "version": "13.0.2.0.0",
```
Comment (simahawk) [inline base_edi/__manifest__.py]: IMO all these changes do not deserve a major release as they are backward compat...

**Review by simahawk — APPROVED** [review]

File: base_edi/__manifest__.py (hunk 1 above)
Comment (etobella) [reply base_edi/__manifest__.py]: True, changed

**Pull request merged**